bin/argo-api-authn-scripts/ams-create-users-gocdb.py

```diff
@@ -10,8 +10,170 @@
 import logging.handlers
 import argparse
 
+# set up logging
+LOGGER = logging.getLogger("AMS User create script")
 
-def create_users(config, logger, verify):
+ACCEPTED_RDNS = ["CN", "OU", "O", "L", "ST", "C", "DC"]
+
+
+class RdnSequence(object):
```

Comment:
Is this going to be treated also as a module file (since the __init__.py on the outside folder and the class here)? If yes better change the filename to something like ams_create_users_gocdb

Reply:
hmmmmmm, i don't think so. As of now, i think we should treat it as an independent script.